tests/system/large/ml/test_pipeline.py

```diff
@@ -660,13 +670,20 @@ def test_pipeline_columntransformer_to_gbq(penguins_df_default_index, dataset_id
     assert one_hot_encoder.max_categories == 100
     assert transformers[0][2] == "species"
 
-    assert transformers[1][0] == "standard_scaler"
-    assert isinstance(transformers[1][1], preprocessing.StandardScaler)
-    assert transformers[1][2] == "culmen_length_mm"
+    assert transformers[1][0] == "label_encoder"
```

Comment:
Interesting, I wonder how BQML orders the transformers. Not the same as they passed in. Most of the time it is fine, but it may be a problem if those are chained. Can you log a bug to track?

Reply:
Logged a bug here: b/301616374